plugins/eslint-plugin-liferay-portal/lib/rules/no-global-fetch.js

```diff
@@ -11,13 +11,34 @@ module.exports = {
 	create(context) {
 		let foundFetchImport = false;
 
+		const isFetchIdentifier = node => {
+			return node.type === 'Identifier' && node.name === 'fetch';
+		};
+
+		const isValidDefaultImport = (source, specifiers) => {
+			return (
+				source.value.endsWith('fetch.es') &&
```

Comment:
How about `'/fetch.es'` just to be safe?

Reply:
I don't see why not 👍